### PR TITLE
ci: add test runner job

### PR DESCRIPTION
**Pull request by jsarzy**
JIRA: CI-18

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -6,7 +6,7 @@
 	url = https://github.com/phoenix-rtos/libphoenix
 [submodule "phoenix-rtos-build"]
 	path = phoenix-rtos-build
-	url = https://github.com/phoenix-rtos/phoenix-rtos-build
+	url = https://github.com/jsarzy/phoenix-rtos-build
```
Comment (jsarzy) [inline .gitmodules:9]: Must be changed before merging.

File: .gitmodules [hunk 2]
```diff
@@ -27,7 +27,7 @@
 	url = https://github.com/phoenix-rtos/phoenix-rtos-posixsrv
 [submodule "phoenix-rtos-tests"]
 	path = phoenix-rtos-tests
-	url = https://github.com/phoenix-rtos/phoenix-rtos-tests
+	url = https://github.com/jsarzy/phoenix-rtos-tests
```
Comment (jsarzy) [inline .gitmodules:30]: Must be changed before merging.

File: .github/actions/phoenix-build/action.yml [hunk 3]
```diff
@@ -35,19 +43,26 @@ inputs:
...
+    default: ''
+    required: false
 
 # action runner
 runs:
   using: 'docker'
   image: 'phoenixrtos/build'
   env:
     TARGET: ${{ inputs.target }}
+    CONSOLE: ${{ inputs.console }}
```
Comment (nalajcie) [inline .github/actions/phoenix-build/action.yml:57]: this is not set anywhere, but I assume it might be in the future

**Review by nalajcie — COMMENTED** [review]
LGTM but not approving before the merge in `phoenix-rtos-test` will happen ;)

File: .github/actions/phoenix-build/action.yml (hunk 3 above)
Comment (jsarzy) [reply .github/actions/phoenix-build/action.yml]: You are right - it is not necessary for now. I removed it

**Comment by nalajcie** [comment]
please update the submodules in this PR so we can merge it ;)

**Comment by jsarzy** [comment]
Done!

**Review by nalajcie — APPROVED** [review]

**Pull request merged**